app/Http/Controllers/AssessmentPlanController.php

```diff
@@ -57,4 +58,16 @@ public function getForJob(JobPoster $jobPoster)
             'rating_guide_answers' => $answers->toArray()
         ];
     }
+
+    public function show(JobPoster $jobPoster)
+    {
+        // Show demo notification if the user is a demoManager and is not an hr-advisor that has claimed the job.
+        $hide_demo_notification = Auth::user() !== null && Auth::user()->isDemoManager() && !$jobPoster->hr_advisors->contains('user_id', Auth::user()->id);
```

Comment:
I think you meant to name this `show_demo_notification`.

I think the logic is not quite right. Right now this returns false if the user is not logged in. Also, the `jobPoster->hr_advisors->contains...` check should be accompanied by a $user->isHrAdvisor()` check, since its possible to be demoted from hr advisor.